GuildedRoseLLC/Constants.swift

```diff
@@ -0,0 +1,3 @@
+public enum StoryboardIDs {
```

Comment:
What was the rationale for introducing a new class for this constant?

Also, a small naming suggestion. If we do decide to keep this class, I would rename it to `StoryboardIds` instead of `StoryboardIDs`.

Reply:
Another naming suggestion: `ItemSegue.showDetails` reads well to me.

Reply:
Very nice job making it an `enum`!